modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -171,6 +171,48 @@ function FieldBase({
 	</>
 	);
 
+	const buttons = repeatable && (
+        <div className="lfr-ddm-form-field-repeatable-toolbar react-field-base-flex-container">
```

Comment:
@mateusgomes01, you've renamed the CSS classes, but this renaming wasn't replicated here. Can you please update and retest?

Reply:
Sorry, I forgot! Gonna fix it and PR again. Thanks!